app/elements/pw-shell.html

```diff
@@ -161,11 +158,11 @@
       }
     </style>
 
-    <app-location route="{{_sectionRoute}}"></app-location>
-    <app-route route="{{_sectionRoute}}" pattern="/:version/:section" data="{{_sectionData}}" tail="{{_subsectionRoute}}"></app-route>
-    <app-route route="{{_sectionRoute}}" pattern="/search/" query-params="{{_searchParams}}"></app-route>
+    <app-location route="{{_route}}" path="{{path}}"></app-location>
```

Comment:
I think this is the only part i find a bit scary because I don't know how app-location really works